rust/debian/Dockerfile

```diff
@@ -7,13 +7,13 @@ ENV SCCACHE_BUCKET=${SCCACHE_BUCKET}
 ARG SCCACHE_REGION
 ENV SCCACHE_REGION=${SCCACHE_REGION}
 
-ARG ARCH=x86_64
```

Comment:
Then ARCH here is the target arch rather than the host architecture, I think we need to update the Makefile to use something like docker buildx to cross compile/build the actual image for the different platforms rather than modifying the target arch logic

Reply:
updated